stdlib/encodings/utf_8_sig.pyi

```diff
@@ -0,0 +1,29 @@
+import codecs
+
+class IncrementalEncoder(codecs.IncrementalEncoder):
+    def __init__(self, errors: str = ...) -> None: ...
+    def encode(self, input: str, final: bool = ...) -> bytes: ...
+    def reset(self) -> None: ...
+    # N.B. Inline ignore these lines for mypy as source code
+    # violates the Liskov Substitution Principle.
+    def getstate(self) -> int: ...  # type: ignore
+    def setstate(self, state: int) -> None: ...  # type: ignore
```

Comment:
You can use `# type: ignore[override]`, which is self-documenting enough that it doesn't need the first comment. Even though the syntax is technically mypy specific, it is just like `# type: ignore` in other type checkers, and we decided to use it in #6228.

Reply:
makes sense thanks, I've updated that now 😄